spec/notification.js

```diff
@@ -375,6 +375,34 @@ describe('notification api', function() {
           });
     });
 
+    it('should get latest 250 received texts', function(done) {
+
+      notifyAuthNock
+        .get('/v2/received-text-messages')
+        .reply(200, {"foo":"bar"});
+
+      notifyClient.getReceivedTexts()
+        .then(function(response){
+          expect(response.statusCode).to.equal(200);
+          done();
```

Comment:
again don't want `done` here. You'll need to `return notifyClient.getReceivedTexts() ...` tho